server/sonar-web/src/main/js/apps/issues/components/FiltersHeader.js

```diff
@@ -21,15 +21,17 @@
 import React from 'react';
 import { translate } from '../../../helpers/l10n';
 
+/*::
 type Props = {
   displayReset: boolean,
   onReset: () => void
 };
+*/
 
 export default class FiltersHeader extends React.PureComponent {
-  props: Props;
+  /*:: props: Props; */
 
-  handleResetClick = (e: Event & { currentTarget: HTMLElement }) => {
+  handleResetClick = (e /*: Event & { currentTarget: HTMLElement } */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)